libs/CCAppCommon/src/ccApplicationBase.cpp

```diff
@@ -273,15 +273,41 @@ void ccApplicationBase::setupPaths()
 
 bool ccApplicationBase::setAppStyle(QString styleKey)
 {
-	QStyle* style = QStyleFactory::create(styleKey);
-	if (!style)
-	{
-		ccLog::Warning("Invalid style key or style couldn't be created: " + styleKey);
-		return false;
-	}
+	const auto loadStyleSheet = [this](const QString& ressourcePath) {
+		QFile f(ressourcePath);
+		if (!f.exists())   {
+			f.close();
+			return false;
+		}
+		else {
+			f.open(QFile::ReadOnly | QFile::Text);
+			QTextStream ts(&f);
+			setStyleSheet(ts.readAll());
+			f.close();
+			return true;
+		}
+	};
+
+	if (styleKey == "QDarkStyleSheet::Dark") {
+		if (!loadStyleSheet(":qdarkstyle/dark/darkstyle.qss")) {
+			return false;
+		}
+	} else if (styleKey == "QDarkStyleSheet::Light") {
+		if (!loadStyleSheet(":qdarkstyle/light/lightstyle.qss")) {
+			return false;
+		}
+	} else {
+		QStyle* style = QStyleFactory::create(styleKey);
+		if (!style)
+		{
+			ccLog::Warning("Invalid style key or style couldn't be created: " + styleKey);
+			return false;
+		}
 
-	ccLog::Print("Applying application style: " + styleKey);
-	setStyle(style);
+		setStyleSheet("");
```

Comment:
```suggestion
		setStyleSheet(QString());
```

or maybe setStyleSheet({}); if it works